scripts/process_data.py

```diff
@@ -420,51 +427,74 @@ def main(
                 num_frames = copy_images(data, image_dir=image_dir, verbose=verbose)
         else:
             num_frames = copy_images(data, image_dir=image_dir, verbose=verbose)
-        CONSOLE.log("[bold green]:tada: Done copying images.")
-        summary_log.append(f"Starting with {num_frames} images")
 
-    if num_downscales > 0:
-        if not verbose:
-            with CONSOLE.status("[bold yellow]Downscaling images...", spinner="growVertical"):
-                downscale_images(image_dir, num_downscales, verbose=verbose)
+        if num_frames == 0:
+            CONSOLE.log("[bold red]No usable images in the data folder.")
         else:
-            downscale_images(image_dir, num_downscales, verbose=verbose)
-        CONSOLE.log("[bold green]:tada: Done downscaling images.")
-        downscale_text = [f"[bold blue]{2**(i+1)}x[/bold blue]" for i in range(num_downscales)]
-        downscale_text = ", ".join(downscale_text[:-1]) + " and " + downscale_text[-1]
-        summary_log.append(f"We downsampled the images by {downscale_text}")
-
-    camera_model = CAMERA_MODELS[camera_type]
-    colmap_dir = output_dir / "colmap"
-    if not skip_colmap:
-        colmap_dir.mkdir(parents=True, exist_ok=True)
-
-        run_colmap(
-            image_dir=image_dir,
-            colmap_dir=colmap_dir,
-            camera_model=camera_model,
-            gpu=gpu,
-            verbose=verbose,
-            matching_method=matching_method,
-        )
+            CONSOLE.log("[bold green]:tada: Done copying images.")
+        summary_log.append(f"Starting with {num_frames} images")
+
+    if num_frames > 0:
+        if num_downscales > 0:
+
+            # Change image_dir for COLMAP to use the one closest to the 2000px longest side
+            # To improve the solve speed
 
-    if (colmap_dir / "sparse" / "0" / "cameras.bin").exists():
-        with CONSOLE.status("[bold yellow]Saving results to transforms.json", spinner="balloon"):
-            num_matched_frames = colmap_to_json(
-                cameras_path=colmap_dir / "sparse" / "0" / "cameras.bin",
-                images_path=colmap_dir / "sparse" / "0" / "images.bin",
-                output_dir=output_dir,
+            max_side = max(Image.open(next(image_dir.glob("[!.]*"))).size)
+            factor = max_side // 2000
```

Comment:
Should 2000 be a parameter someone can change from the terminal? If so, we can document it there.

Reply:
Sure. I forgot this PR is still opened and pushed without consideration. I am still doing tests so I'd rather roll back a commit